### ISSUE TITLE
`tinygo info bluepill` fails in ubunt 20.04 with a user installed Go SDK

### ISSUE THREAD

**Issue by dolanor** [tinygo-org/tinygo#3241]
Hi,

Using the TinyGo VS Code extension, I got a: `Could not run 'tinygo info bluepill': Error: Command failed: tinygo info bluepill requires go version 1.18 through 1.19, got go1.14` error message

I installed Go the traditional way on Ubuntu 20.04 to have it with the latest version (in `/usr/local/go`). It was Go 1.14
After that, I used the https://go.dev/doc/manage-install tutorial and mechanism to update my version of Go: it install in ~/sdk/goX.Y.Z .

TinyGo could not find this version, even though this version of `go` cmd was in my $PATH, preceding `/usr/local/go/bin`. But it kept failing giving me the info.

**Comment by kenbell** [comment]
Have you tried setting GOROOT env variable?  I think this is the code that detects where Go is installed.  It has a bunch of fall-back strategies.

**Comment by aykevl** [comment]
> Have you tried setting GOROOT env variable?

In my experience, this advice usually results in more problems than it solves.

I investigated this issue (long thread on Slack!) but couldn't really figure out what's going on. But it does appear that it doesn't recognize the Go SDK directory format or the way the symlinks are made, or something like that. So when it can't find the directory, it falls back to a fixed default (/usr/local/go).

...but yeah, in this particular case setting a GOROOT could be a temporary solution.

**Comment by dolanor** [comment]
Yes, I haven't used GOROOT since I started working in Go, 8 years ago. It was already obsoleted.
I tried, though, for this issue, and in fact, it works :)

The best, I guess, would be to try `go env GOROOT` instead of `os.Getenv("GOROOT")`
